dev-haskell/cabal-install/cabal-install-3.2.0.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.5.9999
+
+CABAL_FEATURES=""
+inherit haskell-cabal
+
+DESCRIPTION="The command-line interface for Cabal and Hackage"
+HOMEPAGE="http://www.haskell.org/cabal/"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug-conflict-sets debug-expensive-assertions debug-tracetree +native-dns"
+
+RDEPEND=">=dev-haskell/async-2.0:= <dev-haskell/async-2.3:=
+	>=dev-haskell/base16-bytestring-0.1.1:= <dev-haskell/base16-bytestring-0.2:=
+	>=dev-haskell/cabal-3.2:= <dev-haskell/cabal-3.3:=
+	>=dev-haskell/cryptohash-sha256-0.11:= <dev-haskell/cryptohash-sha256-0.12:=
+	>=dev-haskell/echo-0.1.3:= <dev-haskell/echo-0.2:=
+	>=dev-haskell/edit-distance-0.2.2:= <dev-haskell/edit-distance-0.3:=
+	>=dev-haskell/fail-4.9:= <dev-haskell/fail-4.10:=
+	>=dev-haskell/hackage-security-0.6.0.0:= <dev-haskell/hackage-security-0.7:=
+	>=dev-haskell/hashable-1.0:= <dev-haskell/hashable-1.4:=
+	>=dev-haskell/http-4000.1.5:= <dev-haskell/http-4000.4:=
+	>=dev-haskell/lukko-0.1:= <dev-haskell/lukko-0.2:=
+	>=dev-haskell/mtl-2.0:= <dev-haskell/mtl-2.3:=
+	>=dev-haskell/network-2.6:= <dev-haskell/network-3.2:=
+	>=dev-haskell/network-uri-2.6.0.2:= <dev-haskell/network-uri-2.7:=
+	>=dev-haskell/parsec-3.1.13.0:= <dev-haskell/parsec-3.2:=
+	>=dev-haskell/random-1:= <dev-haskell/random-1.2:=
+	>=dev-haskell/semigroups-0.18.3:= <dev-haskell/semigroups-0.20:=
+	>=dev-haskell/stm-2.0:= <dev-haskell/stm-2.6:=
+	>=dev-haskell/tar-0.5.0.3:= <dev-haskell/tar-0.6:=
+	>=dev-haskell/text-1.2.3:= <dev-haskell/text-1.3:=
+	>=dev-haskell/zlib-0.5.3:= <dev-haskell/zlib-0.7:=
+	>=dev-lang/ghc-7.10.1:=
+	debug-tracetree? ( >=dev-haskell/tracetree-0.1:= <dev-haskell/tracetree-0.2:= )
+	native-dns? ( >=dev-haskell/resolv-0.1.1:= <dev-haskell/resolv-0.2:= )
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-1.22.2.0
+"
+
```

Comment:
we need a `src_prepare()` phase here to allow `base-4.14` with `cabal_chdeps`, and it's probably worth copying over the `src_prepare()` stuff from the previous version too